### ISSUE TITLE
SVG images are inheriting external css styles

### ISSUE THREAD

**Issue by maikuru** [AnalyticalGraphicsInc/cesium#2785]
If you have an svg loaded on the external DOM and used within Cesium it appears the css property is overriding the Cesium property.
 
https://groups.google.com/forum/#!msg/cesium-dev/GEvwmTwXhz4/-YaBo7zQV7kJ

**Comment by mramato** [comment]
This doesn't happen in Firefox and Cesium is not do anything special other than to ask the browser to load the SVG image.  So it's either a bug in Firefox (for not properly taking the class into account) or a bug in Chrome (for styling all svgs when one of them has an assigned class).  Looking at the markup, I think the bug is in Chrome.  In either case, I don't believe this is a bug in Cesium.

@emackey any thoughts?  If you agree with me, please close this out.  I would recommend someone report the issue to Chromium (if it's then) or Mozilla otherwise.

**Comment by maikuru** [comment]
It's happening for me in Chrome.

**Comment by emackey** [comment]
Yeah this does look like a Chrome bug.  Changing the class name on the `img` tag changes the behavior in Cesium, which is crazy.

**Comment by emackey** [comment]
Making a reduced test case could take some effort, I would imagine...

**Comment by mramato** [comment]
@maikuru, that's exactly what I said.  It only happens in Chrome.

**Comment by mramato** [comment]
> Changing the class name on the img tag changes the behavior in Cesium, which is crazy.

I can't reproduce that.  If I change `pic` to `foo` everywhere, the behavior is the same.  If I change `img.pic` to just `pic`, it works as expected.  Maybe a selector issue?

**Comment by mramato** [comment]
Nevermind, just `pic` still has the problem, I forgot the `.`

**Comment by pjcozzi** [comment]
OK to close this?
